script/legacy/2019-05-14-armagaddon/delete-from-bq.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+## Affected period to delete is
+## 2019-05-04T11:00:00Z to 2019-05-11T11:00:00Z
+
+function delete_partitions() {
+    local TABLE=$1
+    echo bq rm -f -t "$TABLE"'$20190504'
+    echo bq rm -f -t "$TABLE"'$20190505'
+}
+
+## Tables populated by Dataflow jobs
+delete_partitions moz-fx-data-shar-nonprod-efed.activity_stream.impression_stats_v1
+delete_partitions moz-fx-data-shar-nonprod-efed.activity_stream.spoc_fills_v1
+
+## Derived tables specific to GCP
+delete_partitions moz-fx-data-derived-datasets.clients_last_seen_v1
```

Comment:
We'll need some additional nuance here since clients_last_seen depends on 28 days of history. If we only delete these days, we'll still have data leaking, since we've already carried over some 2019-05-11 values into the 2019-05-12 partition that we're keeping.

Reply:
we should remove the days from clients_daily and then backfill the table starting with the first day deleted

Reply:
We're potentially going to have some more discussion with data science around this, since MAU calculated from clients_last_seen is going to be affected for a full 28 days after the end of the deleted region.